Yoast/wordpress-seo
# Unescape replacement variable string values

## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where the conversion of `&#039;`, which is PHP's HTML entity for the apostrophy, did not happen.

## Relevant technical choices:

* Lodash's unescape function only escapes the `&#39;` HTML identity, but PHP's escapes apostrophes with `&#039;` which was not captured by Lodash's RegEx. I have wrapped unescape in a helper function that will take the `&#039;` into account.

## Test instructions

This PR can be tested by following these steps:

* Go to your blog's settings and set a `Site Title` that contains an apostrophe.
* Previously, when using the `Site title` replacement variable, it will fill in your blog's name, but the apostrophy would be replaced by `&#039;`. Verify that this is now fixed.

## Quality assurance


Fixes #10185
